bom/runtime/pom.xml

```diff
@@ -1976,11 +1985,36 @@
                 <artifactId>infinispan-commons</artifactId>
                 <version>${infinispan.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.infinispan</groupId>
+                <artifactId>infinispan-lucene-directory</artifactId>
```

Comment:
Is there a reason we are including lucene and directory providers here? These are server artifacts that aren't going to be supported in Quarkus native mode since lucene is not supported there.

Reply:
@wburns added it here so that the correct versions are enforced in _quarkus-integration-test-infinispan-client_. I believe these deps get pulled in order to run the _InfinispanServerTestResource_.

Reply:
@wburns are the answers OK for you?

Reply:
Seems okay to me.